docs/CodeDesign.md

```diff
@@ -8,8 +8,11 @@ project and immediately gets some useful and actionable information.
 
 ## Boundaries
 
-- Support all current Python versions: that means all Python versions that have
-  a stable release, and are not yet End Of Life. Currently this is: v3.7 - v3.11.
+- Support all current Python versions: that means [all Python versions that have
+  a stable release, and are not yet End Of Life](https://devguide.python.org/versions/).
+  - Currently we support running on Python v3.8 - v3.13.
+  - Since we no longer rely on running inside the same Python environment as the project being
+    analyzed, it is possible for us to support analyzing projects running on even older Python versions.
```

Comment:
Same problem with mkdocs list rendering

```suggestion
- Support all current Python versions: that means [all Python versions that have
  a stable release, and are not yet End Of Life](https://devguide.python.org/versions/).
    - Currently we support running on Python v3.8 - v3.13.
    - Since we no longer rely on running inside the same Python environment as the project being
      analyzed, it is possible for us to support analyzing projects running on even older Python versions.
```